src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3687,6 +3692,33 @@ def should_enable_distributed_tracing(consumption_plan_location, matched_runtime
         and matched_runtime.name.lower() == "java" \
         and image is None
 
+def update_functionapp_polling(cmd, resource_group_name, name, functionapp):
```

Comment:
Copied this function from @kamperiadis's PR at https://github.com/Azure/azure-cli/pull/27589. We both are doing a similar change.